tester_app/client/app/bundles/HelloWorld/components/HelloWorldWidget.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { PropTypes } from 'react';
+import _ from 'lodash';
+
+// Simple example of a React "dumb" component
+export default class HelloWorldWidget extends React.Component {
+  constructor(props, context) {
+    super(props, context);
+
+    // Uses lodash to bind all methods to the context of the object instance, otherwise
+    // the methods defined here would not refer to the component's class, not the component
+    // instance itself.
+    _.bindAll(this, "_handleChange");
+  }
+
+  static propTypes = {
+    name: PropTypes.string.isRequired,
+    updateName: PropTypes.func.isRequired,
+  };
+
+  // React will automatically provide us with the event `e`
+  _handleChange(e) {
+    const name = e.target.value;
+    this.props.updateName(name);
+  }
+
+  render() {
+    const _handleChange = this._handleChange;
```

Comment:
@justin808 you told me to do it Alex's way where he just does it no matter what: https://github.com/shakacode/react_on_rails-generator-results-pre-0/pull/6#discussion_r43587922